### DISCUSSION TITLE
Update iOS shortcut guide

### DISCUSSION THREAD

**Discussion by rastacalavera** [mealie-recipes/mealie#4531] [General]
Are there community members out there who would be willing to document and update the iOS shortcut guide? I know it’s a small group that maintains the actual shortcut but it would be nice to see an article about the pros/cons of using the gemni api vs Mealies OpenAI integration now and perhaps offering a shortcut for each option.

**Comment by ChipSkylark37** [comment]
Hey there -

I'd be willing to write something up, though I've not setup the iOS shortcut where it's actually working yet. I'll try the AI method as another way to do this. 

Either way, I don't mind creating / updating a guide once I get it working.

For the shortcut:
My instance is exposed publicly but through a reverse tunnel with cloudflare. So when I run the shortcut I get stuck on the first step. I'll do some troubleshooting on this, and typically it requires setup as a service principal. 

I'm hoping I can get it to work as this would be helpful in either method.

**Comment by Kuchenpirat** [comment]
since the shortcut is no longer maintained we have decided to remove it (#5280)
